package.json

```diff
@@ -193,7 +186,7 @@
     ]
   },
   "prettier": {
-    "printWidth": 120
```

Comment:
any specific reason to change thing? asking cause all our repo using that setting i think

Reply:
haha this one no much reason cos previously i was using 100 and i think it look pretty when having 2 files side by side. haha i can change it back.

Reply:
Ok fixed the validation.